tests/acceptance/features/api/api_sync_region.feature

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+Feature: Synchronize the images of the corresponding region defined by its regionId.
+         The operation is asynchronous it responses with a taskId in order that
+         you can follow the execution of the process.
+
+  As a GlanceSync API user
+  I want to sync images in a region using the GlanceSync API
+  So that I can have all FIWARE images up to date in all FIWARE nodes.
+
+  GlanceSync API operation under testing:
+    POST regions/regionId
+
+
+  @happy_path @env_dependant @experimentation @skip # needs a real environment.
```

Comment:
In this scenario, I am using a explicit dataset (name of the region) that is particular of our Experimentation environment. This scenario, as it has been described, will not be able to run on other environment if it does not have the same name of regions

         @env_dependant @experimentation

This scenario is a "Happy Path": a default scenario featuring no exceptional or error conditions, and comprises the sequence of activities executed if everything goes as expected. The tag is very useful to execute only these type of scenarios as part of Smoke Tests or Sanity Checks for the component.

          @happy_path